src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -100,31 +104,39 @@ const ProfileHeader = ({
         >
           {showBadgeImage && (
             <img
-              className="profile-service-badge"
               src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
-              alt="service badge"
+              alt={`${latestBranchOfService} seal`}
+              className="profile-service-badge"
+              aria-hidden="true"
+              role="presentation"
             />
           )}
         </div>
         <div className="name-and-title-wrapper">
           <h1 className={[...titleClasses, ...titleClassesMedium].join(' ')}>
             Your Profile
           </h1>
-          <h2
-            className={[...fullNameClasses, ...fullNameClassesMedium].join(' ')}
-          >
-            {fullName}
-          </h2>
-          {latestBranchOfService && (
-            <h3
-              className={[
-                ...latestBranchClasses,
-                ...latestBranchClassesMedium,
-              ].join(' ')}
+          <dl className="vads-u-margin-y--0">
```

Comment:
We are now wrapping the name and service branch in a `dl`. Which unfortunately introduces some diff noise due to the change in indentation.